### DISCUSSION TITLE
Neuron Compounding

### DISCUSSION THREAD

**Discussion by BirdZiggy** [nervosnetwork/neuron#2492] [Ideas]
Hello everyone, I'm relatively new to the Nervos space but enjoying the community and have been using Neuron wallet. Not sure if this has been brought up, or if I have missed a setting, but it would be awesome to have a compounding option within the DAO in which after the compensation cycle the CKB rewards are added to the initial DAO deposit amount and then APC applied to that amount. 

Currently it appears that you need to withdraw everything and then resubmit manually in order to compound your rewards in the DAO. Since some people may not want that a toggle feature would be sweet. 

Thoughts? 

Let's build a new more fair world together! 
BirdZiggy

**Comment by yanguoyu** [comment]
I'm confirming from the DAO deviser whether the DAO has considered the compound interest.

**Reply by yanguoyu** [reply]
@BirdZiggy The DAO has considered the compound interest. So you just keep the `CKB` in DAO is ok.

**Reply by BirdZiggy** [reply]
awesome! I'll keep it where it is. looking forward to further development and being part of this chain's growth.

**Comment by EthAnalysis** [comment]
I think I noticed that the APY decreases the next time you would withdraw/and resubmit; so essentially keeping it in the DAO is like it compounding. DAO designers accounted for this as @yanguoyu has pointed out. Just confirming.
